src/main/webapp/onelive/archive/index.html

```diff
@@ -100,9 +100,17 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
             url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
             dataType: 'json',
             success: function (data) {
+                let newData = data.items;
+                //check description length
```

Comment:
- [ ] Add a meaningful comment.  
- [ ] Also start with a capital letter and add space before the sentence. 

ex //<space>Lorem ipsum